docs/appendices/release-notes/unreleased.rst

```diff
@@ -48,7 +48,8 @@ None
 Deprecations
 ============
 
-None
+- The settings ``translog.retention.age``and ``translog.retention.size`` have
```

Comment:
We have never exposed and documented these settings so we don't need to document the deprecations.